tests/v2/validation/provisioning/rke1/delete_rke1_node_driver_cluster_test.go

```diff
@@ -0,0 +1,238 @@
+	session            *session.Session
+	standardUserClient *rancher.Client
+	kubernetesVersions []string
+	cnis               []string
+	providers          []string
+	nodeProviders      []string
+}
+
+func (r *RKE1NodeDriverDeletingTestSuite) TearDownSuite() {
+	r.session.Cleanup()
+}
+
+func (r *RKE1NodeDriverDeletingTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	r.session = testSession
+
+	clustersConfig := new(provisioning.Config)
+	config.LoadConfig(provisioning.ConfigurationFileKey, clustersConfig)
+
+	r.kubernetesVersions = clustersConfig.RKE1KubernetesVersions
+	r.cnis = clustersConfig.CNIs
+	r.providers = clustersConfig.Providers
+	r.nodeProviders = clustersConfig.NodeProviders
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(r.T(), err)
+
+	r.client = client
+
+	enabled := true
+	var testuser = namegen.AppendRandomString("testuser-")
+	var testpassword = password.GenerateUserPassword("testpass-")
+	user := &management.User{
+		Username: testuser,
+		Password: testpassword,
+		Name:     testuser,
+		Enabled:  &enabled,
+	}
+
+	newUser, err := users.CreateUserWithRole(client, user, "user")
+	require.NoError(r.T(), err)
+
+	newUser.Password = user.Password
+
+	standardUserClient, err := client.AsUser(newUser)
+	require.NoError(r.T(), err)
+
+	r.standardUserClient = standardUserClient
+}
+
+func (r *RKE1NodeDriverDeletingTestSuite) CreateAndDeleteRKE1Cluster(t *testing.T, client *rancher.Client, provider Provider, nodesAndRoles []nodepools.NodeRoles, kubeVersion, cni string, nodeTemplate *nodetemplates.NodeTemplate, externalNodeProvider *provisioning.ExternalNodeProvider) (*management.Cluster, error) {
+	nodeNames := []string{}
+	clusterName := namegen.AppendRandomString(provider.Name.String())
+	cluster := clusters.NewRKE1ClusterConfig(clusterName, cni, kubeVersion, client)
+	clusterResp, err := clusters.CreateRKE1Cluster(client, cluster)
+	require.NoError(t, err)
+
+	nodePool, err := nodepools.NodePoolSetup(client, nodesAndRoles, clusterResp.ID, nodeTemplate.ID)
+	require.NoError(t, err)
+
+	nodePoolName := nodePool.Name
+
+	opts := metav1.ListOptions{
+		FieldSelector:  "metadata.name=" + clusterResp.ID,
+		TimeoutSeconds: &defaults.WatchTimeoutSeconds,
+	}
+
+	adminClient, err := rancher.NewClient(client.RancherConfig.AdminToken, client.Session)
+	require.NoError(t, err)
+	watchInterface, err := adminClient.GetManagementWatchInterface(management.ClusterType, opts)
+	require.NoError(t, err)
+
+	checkFunc := clusters.IsHostedProvisioningClusterReady
+
+	err = wait.WatchWait(watchInterface, checkFunc)
+	require.NoError(t, err)
+	assert.Equal(t, clusterName, clusterResp.Name)
+	assert.Equal(t, nodePoolName, nodePool.Name)
+	assert.Equal(t, kubeVersion, clusterResp.RancherKubernetesEngineConfig.Version)
+
+	clusterIDName, err := clusters.GetClusterIDByName(r.client, clusterName)
+	assert.NoError(r.T(), err)
+
+	fmt.Println("--------Checking if the nodes are ready-----------", clusterIDName)
```

Comment:
No print statements at all, they should either be `logrus.Info` or` r.T().Logf()`

Reply:
I 100% agree, please reference to the PR guidelines